### ISSUE TITLE
Question about the checkpoint provided in this repo

### ISSUE THREAD

**Issue by StarDewXXX** [uclaml/SPIN#28]
Is the checkpoint provided in this repo trained with the old version of zephyr-7b-sft-full?

**Comment by StarDewXXX** [comment]
if I want to reproduce the results reported in SPIN's paper, should I use revision=ac6e600eefcce74f5e8bae1035d4f66019e93190 and train with the SPIN dataset(iter 0,1,2,3) provided in huggingface?

**Comment by angelahzyuan** [comment]
Yes, and we have updated the scripts and guidelines to reproduce the results in our paper. We specified the version by adding model_revision parameter in config.yaml. Please check our recent update and let us know if there are any remaining questions.

**Comment by angelahzyuan** [comment]
> if I want to reproduce the results reported in SPIN's paper, should I use revision=ac6e600eefcce74f5e8bae1035d4f66019e93190 and train with the SPIN dataset(iter 0,1,2,3) provided in huggingface?

https://github.com/uclaml/SPIN/blob/main/configs/config.yaml
